.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Would be nice if the `rc` files were fields in `package.json` versus individual files since they start to clutter up the root of the project 😞

Reply:
OK didn’t know we have a preference like that - Will make a change there!